src/components/MoreInfo/MoreInfoImages/MoreInfoImages.component.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {ImgContaiter, MainImg, MainImgWrap, OtherImgs, SmallImg, SmallImgWrap} from "../MoreInfo.style";
+
+const SmallImages = ({image, product, openPhotosModal, setImageId, imageId}) => {
+
+    const openPhotos = (product, imageId) => {
+        openPhotosModal(product)
+        setImageId(imageId)
+    }
+
+    return <SmallImgWrap>
+        <SmallImg src={image} onClick={() => openPhotos(product, imageId)}/>
+    </SmallImgWrap>
+}
+
+const MainImage = ({image, product, openPhotosModal, setImageId}) => {
+
+    const openPhotos = (product, imageId) => {
+        openPhotosModal(product)
+        setImageId(imageId)
+    }
+
+    return <MainImgWrap>
+        <MainImg src={image} onClick={() => openPhotos(product, 0)}/>
+    </MainImgWrap>
+}
+
+const MoreInfoImages = ({openPhotosModal, product, setImageId}) => {
+    return (
+            <ImgContaiter>
+                <MainImage image={product.image}
+                           setImageId={setImageId}
+                           openPhotosModal={openPhotosModal}
+                           product={product}/>
+
+                {Array.isArray(product.image)
```

Comment:
Can be
{**!**Array.isArray(product.image) 
? null
:

Reply:
understood